lib/web_of_science/harvester.rb

```diff
@@ -10,21 +10,30 @@ class Harvester < ::Harvester::Base
     # @param [Enumerable<Author>] authors
     # @return [void]
     def harvest(authors)
+      logger.info "#{self.class} - started harvest(authors) - #{authors.count} authors"
```

Comment:
To be clear, this `.count` will issue a new SQL query.

Reply:
It's a one-off and quick query, I assume.  For logs, it's a sanity check on how many authors are going to be harvested.  Since that number is dynamic, it's useful log data.  It's also a useful before/after sanity check on the success rate for the harvest.

Reply:
Yeah, not objecting, just making sure we were on the same page.